templates/travel/partials/like/_like.css

```diff
@@ -0,0 +1,158 @@
+/**
```

Comment:
all css should go to the CSS directory, please fix from previous PR as well

Reply:
Done (this PR)